de.fu_berlin.inf.dpp/test/resources/launch/Saros_STF_Alice.launch

```diff
@@ -13,9 +13,10 @@
 <booleanAttribute key="default" value="true"/>
 <booleanAttribute key="includeOptional" value="true"/>
 <stringAttribute key="location" value="${workspace_loc}/../workspace-alice-stf"/>
+<stringAttribute key="org.eclipse.jdt.launching.JRE_CONTAINER" value="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
```

Comment:
Why ?

Reply:
This line is added by eclipse 4.6 if I execute the launch configuration.